components/fieldgroups/experience-event/experienceevent-campaign-delivery-log.schema.json

```diff
@@ -0,0 +1,138 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/experienceevent-campaign-delivery-log",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Campaign Delivery Logs",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Campaign Delivery Events such as sent, bounces, failures.",
+  "definitions": {
+    "experienceevent-campaignlog": {
+      "properties": {
+        "xdm:directMarketing": {
+          "type": "object",
+          "description": "Captures direct marketing related details.",
+          "properties": {
+            "xdm:externalId": {
+              "title": "External Identifier",
+              "type": "string",
+              "description": "External Identifier that defines the primary identity value for profile and audience."
+            },
+            "xdm:campaignId": {
+              "title": "Campaign Identifier",
+              "type": "string",
+              "description": "Identifier of the marketing campaign to which activity originating this message belongs to."
+            },
+            "xdm:campaignName": {
```

Comment:
This will introduce a directMarketing/campaignName where we already have a marketing/campaignName (from the marketing field group). When should I use one over the other?

Reply:
Hi Craig

Reply:
The marketing field group is defined here:
https://github.com/adobe/xdm/blob/master/components/fieldgroups/experience-event/experienceevent-marketing.schema.json
You can see what it looks like in it's fully resolved form here:
https://opensource.adobe.com/xdmVisualization/prod/master/fieldgroups.experience-event.experienceevent-marketing.html

Note, a field group is used to define fields that can be added to a schema. This PR is introducing a "new" field group (Campaign Delivery Logs) that can be used to add fields to a schema. I want to make sure there is no confusion for a customer when the schema has the potential to contain both a marketing/campaignName (coming from the marketing field group) and this new directMarketing/campaignName (coming from this new field group).

Reply:
I agree that there will be a confusion around this name of the fields.  It is possible that both may have the same meaning and intent but the one we are defining is for a specific solution.  If there is an overlap should we define this a solution specific namespace then.

Reply:
@jishah-adobe - Is it really necessary to have the campaignName here? I'm assuming the actual campaign details would be in another table where you could lookup the current name associated with that campaign.

Reply:
There is a specific use case where customers may want to add filters on campaign names when doing segmentation queries.  The ID that we store are internal ID and customers don't know about them.  Customers know about 2 fields (Name and Label).  Name is Unique across campaigns (alternate unique key) while labels are not unique but are set by customers and they follow some naming conventions.  So both are necessary to ensure audience segmentation can be done based on the either name or label.  Also from discussion with ACC Engineering Team Campaign Name never gets changed (not edited). Label Modifications is also rare.